source/includes/api/_geostore.md

```diff
@@ -1,58 +1,71 @@
 # Geostore
 
-## What is Geostore?
+***Note — This is the documentation for version 1 of the Geostore microservice, the latest version is [version 2](https://resource-watch.github.io/doc-api/index-rw.html#geoStore-v2).***
 
-Geostore is a geojson database. You can save your geojson or obtain geojson by country, region, Protected areas, etc.
+## What is Geostore
 
-Geostore object contains the following fields:
+Geostore is a database for storing geoStore objects which contain geographic data structures described using the [GeoJSON](https://geojson.org/) schema. GeoJSON supports the following geometry types: `Point`, `LineString`, `Polygon`, `MultiPoint`, `MultiLineString`, and `MultiPolygon`. Geometric objects with additional properties are `Feature` objects. Sets of features are contained by `FeatureCollection` objects.
 
+The Geostore microservice provides a comprehensive way to communicate geographic data structures between platforms and microservices. It enables users to [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) geographic data structures from the Geostore, and query information about a geoStore object, such as the bounding box of all its' geometries.
```

Comment:
I get the idea between using the CRUD term, but I don't think it applies here - I think you cant delete geostores, and updates are limited AFAIK. I think this may ultimately be misleading to readers

Reply:
Changed this section to be simpler and more readable. Removed reference to CRUD.